atom/common/api/atom_api_native_image.cc

```diff
@@ -221,6 +237,50 @@ std::string NativeImage::ToDataURL() {
   return data_url;
 }
 
+v8::Local<v8::Value> NativeImage::AsNativeRepresentation(
+    v8::Isolate* isolate,
+    mate::Arguments* args) {
+  NativeRepresentation desiredRep = NativeRepresentation::INVALID;
+  void* ptr = nullptr;
+  std::string type;
+
+  if (!args->GetNext(&type)) {
+    args->ThrowError();
```

Comment:
This case crashes the renderer, but debugging it on OS X doesn't seem to help me much, I just get a bogus stack